services/voice_service/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# syntax=docker/dockerfile:experimental
+
+FROM python:3.7
+
+WORKDIR /src
+
+ARG SERVICE_PORT
+ENV SERVICE_PORT ${SERVICE_PORT}
+
+ARG SERVICE_NAME
+ENV SERVICE_NAME ${SERVICE_NAME}
+
+ARG FLASK_APP
+ENV FLASK_APP ${FLASK_APP}
```

Comment:
remove this two lines. You provide FLASK_APP as environment already in docker-compose. So, here you actually can overwrite this value

Reply:
removed